docs/reference/cryptographic-overview.md

```diff
@@ -0,0 +1,60 @@
+# GitHub runner cryptographic overview
+This document provides an overview of the cryptographic technologies used in the GitHub runner charm, including encryption/decryption, hashing, and digital signatures.
```

Comment:
Two nitpicks on this sentence: 
1. I think you can separate encryption and decryption with a comma 
2. The style guide recommends not using the Oxford comma, so remove the comma after "hashing"